### ISSUE TITLE
Adding rows to a Table doesn't copy calculated column formulas

### ISSUE THREAD

**Issue by craigbrown** [EPPlusSoftware/EPPlus#422]
If I add rows to a table with a "calculated column formula" in one of the columns, the new rows will not include this formula. I believe expected behaviour is that they should include the formula.

**Unit Test**
```csharp
[TestMethod]
public void AddRowsToTableWithCalculatedColumnFormula()
{
    using (var pck = OpenTemplatePackage("TestTableFormulas.xlsx"))
    {
        var sheet1 = TryGetWorksheet(pck, "Sheet1");
        var table1 = sheet1.Tables["Table1"];

        // Check there is a calculated formula in the first column
        var calculatedColumnFormula = table1.Columns[0].CalculatedColumnFormula;
        Assert.IsFalse(string.IsNullOrEmpty(calculatedColumnFormula));

        // Check this formula is present in this column in the first row
        var firstCell = table1.DataRange.Start;
        var firstRow = firstCell.Row;
        var firstColumn = firstCell.Column;
        Assert.AreEqual(calculatedColumnFormula, sheet1.Cells[firstRow, firstColumn].Formula);

        // Add some rows
        table1.AddRow(10);

        // Check the calculated formula is still in the first column
        Assert.AreEqual(calculatedColumnFormula, table1.Columns[0].CalculatedColumnFormula);

        // Check this formula is present in one of the new rows
        var lastRow = table1.DataRange.End.Row;
        Assert.AreEqual(calculatedColumnFormula, sheet1.Cells[lastRow, firstColumn].Formula);
    }
}
```
[TestTableFormulas.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6716802/TestTableFormulas.xlsx)

**Comment by JanKallman** [comment]
Sounds correct. I'll have a look.

**Comment by JanKallman** [comment]
Fixed in version 5.7.1
